src/redux/actions/samples/createSample.js

```diff
@@ -60,9 +62,13 @@ const createSample = (
     if (type === '10X Chromium') {
       sampleTechnology = '10x';
     } else {
-      throw new Error(`Sample type ${type} is not implemented`);
+      throw new Error(`Sample technology ${type} is not recognized`);
     }
 
+    filesToUploadV2Only.forEach((fileName) => {
+      newSample.files[fileName] = { upload: { status: UploadStatus.UPLOADING } };
+    });
```

Comment:
Why do we need this? I see that it is not passed into the request `body`

Reply:
it's so that we know which files to show as `FILE_NOT_FOUND`, without this we would end up showing all files as `FILE_NOT_FOUND` until they are loaded, in this way we are already showing "Uploading" for any file that is already being loaded and `FILE_NOT_FOUND` for any files that are really not there.

There was a different workaround for this in with `processUploadV1` which was way less clear and forced us to create the samples in a weird timing, with this other implementation we are explicitly saying which files exist when we create the sample.